canopeum_frontend/src/components/analytics/AnalyticsSiteHeader.scss

```diff
@@ -15,7 +14,7 @@
 }
 
 @media screen and (min-width: $large-width) {
-  .header-card {
+  .header-card.header-card {
```

Comment:
TODO: Validate if doubling is still needed now that we don't re-import global style